readinglist/util.py

```diff
@@ -0,0 +1,17 @@
+import ast
```

Comment:
#24 will create a utils.py file, maybe we should rename this utils.py as well to share the same file.